dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/timeout/exception/PersistTimeoutException.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.http.netty.timeout.exception;
+
+import java.util.concurrent.TimeUnit;
+
+public class PersistTimeoutException extends TimeoutException{
+
+    private static final long serialVersionUID = 1L;
+    private static final String warningCode = "SOMECODE";
```

Comment:
We need an issue opened to clean up this `SOMECODE` in these exception.

Reply:
We already have one open